mmdet/models/backbones/resnet.py

```diff
@@ -308,10 +318,8 @@ def __init__(self,
                  norm_eval=True,
                  dcn=None,
                  stage_with_dcn=(False, False, False, False),
-                 gcb=None,
-                 stage_with_gcb=(False, False, False, False),
-                 gen_attention=None,
-                 stage_with_gen_attention=((), (), (), ()),
+                 plugin=None,
+                 stage_with_plugin=(False, False, False, False),
```

Comment:
In this way, all plugins share the same setting, which is not flexible if we have multiple plugins at the same time.

It is more flexible (but also complicated) with this type:
```python
plugin = dict(
    conv1=[
        dict(
            cfg=dict(type='xxx', arg1='xxx'), stages=(False, True, True, True))
    ],
    conv2=[
        dict(cfg=dict(type='yyy')),
        dict(cfg=dict(type='yy2'), stages=(False, False, True, True))
    ])
```